rtengine/LibRaw.cmake

```diff
@@ -67,47 +85,44 @@ endif()
 
 # Build flags.
 set(LIBRAW_MAKE_FLAGS "")
-if(CMAKE_GENERATOR MATCHES ".*Makefiles.*")
-    set(LIBRAW_MAKE_COMMAND "$(MAKE)")
-else()
```

Comment:
Please make it possible to use the jobserver. It is very inconvenient for developers otherwise.

Reply:
I didn't take away parallel builds, only jobserver.  So it will continue to be fast, assuming there is enough RAM.

Reply:
I checked things a little bit, so the cause for the Linux build issues I'd observed in the past with Makefiles is a missing dependency between rtengine and "LibRaw", the target that gets defined to "ALL" and to depend on a non-existing phantom file.

That's one part.

Still we need to cater for incompatible "nmake" and other garbage in $(MAKE) and just use gmake or make in that case.  I've pused new material to my branch.

Reply:
My point about the jobserver is that it allows better control over the parallelism. That not only includes increasing the parallelism for faster builds, but also reducing it to keep the rest of the system reasonably responsive.

Reply:
I'm on a laptop with 2/4 physical/logical cores. `-j3` keeps the system responsive while I work on other tasks, at least on Linux. I need to reduce the parallelism even more on Windows. Otherwise it becomes sluggish.